db/db_dungeons.lua

```diff
@@ -3,100 +3,86 @@ local _, ns = ...
+        ["id"] = 14063,
+        ["keystone_instance"] = 399,
+        ["instance_map_id"] = 2521,
+        ["lfd_activity_ids"] = { 1173, 1174, 1175, 1176 },
+        ["timers"] = { 1080, 1440, 1800 },
+        ["name"] = "Ruby Life Pools",
+        ["shortName"] = "RLP",
     },
     [6] = {
-        ["id"] = 999999, -- id
-        ["keystone_instance"] = 234, -- map_challenge_mode_id
-        ["instance_map_id"] = 1651, -- wow_instance_id
-        ["lfd_activity_ids"] = { 472, 473 }, -- https://wow.tools/dbc/?dbc=groupfinderactivity
-        ["timers"] = { 1260, 1680, 2100 }, -- keystone_timer_seconds * 0.6 , keystone_timer_seconds * 0.8, keystone_timer_seconds
-        ["name"] = "Return to Karazhan: Upper",
-        ["shortName"] = "UPPR",
+        ["id"] = 13982,
+        ["keystone_instance"] = 400,
+        ["instance_map_id"] = 2516,
+        ["lfd_activity_ids"] = { 1181, 1182, 1183, 1184 },
+        ["timers"] = { 1440, 1920, 2400 },
+        ["name"] = "The Nokhud Offensive",
+        ["shortName"] = "NO",
     },
     [7] = {
-        ["id"] = 1000000,
-        ["keystone_instance"] = 391,
-        ["instance_map_id"] = 2441,
-        ["lfd_activity_ids"] = { 1016, 1018 },
-        ["timers"] = { 1404, 1872, 2340 },
-        ["name"] = "Tazavesh: Streets of Wonder",
-        ["shortName"] = "STRT",
+        ["id"] = 13954,
+        ["keystone_instance"] = 401,
+        ["instance_map_id"] = 2515,
+        ["lfd_activity_ids"] = { 1177, 1178, 1179, 1180 },
+        ["timers"] = { 1224, 1632, 2040 },
+        ["name"] = "The Azure Vault",
+        ["shortName"] = "AV",
     },
     [8] = {
-        ["id"] = 1000001,
-        ["keystone_instance"] = 392,
-        ["instance_map_id"] = 2441,
-        ["lfd_activity_ids"] = { 1017, 1019 },
-        ["timers"] = { 1080, 1440, 1800 },
-        ["name"] = "Tazavesh: So'leah's Gambit",
-        ["shortName"] = "GMBT",
+        ["id"] = 14032,
+        ["keystone_instance"] = 402,
+        ["instance_map_id"] = 2526,
+        ["lfd_activity_ids"] = { 1157, 1158, 1159, 1160 },
+        ["timers"] = { 1152, 1536, 1920 },
+        ["name"] = "Algeth'ar Academy",
+        ["shortName"] = "AA",
     }
 }
 
 -- Raid listing sorted by id
 ns.raids = {
-	[1] = {
-		["id"] = 13224,
-		["instance_map_id"] = 2296,
-		["lfd_activity_ids"] = { 720, 721, 722 },
-		["name"] = "Fated Castle Nathria",
-		["shortName"] = "FCN",
-	},
-	[2] = {
-		["id"] = 13561,
-		["instance_map_id"] = 2450,
-		["lfd_activity_ids"] = { 743, 744, 745 },
-		["name"] = "Fated Sanctum of Domination",
-		["shortName"] = "FSOD",
-	},
-	[3] = {
-		["id"] = 13742,
-		["instance_map_id"] = 2481,
-		["lfd_activity_ids"] = { 1020, 1021, 1022 },
-		["name"] = "Fated Sepulcher of the First Ones",
-		["shortName"] = "FSFO",
-	}
+    [1] = {
+        ["id"] = 14030,
+        ["instance_map_id"] = 2522,
+        ["lfd_activity_ids"] = { 1189, 1190, 1191 },
+        ["name"] = "Vault of the Incarnates",
+        ["shortName"] = "VOTW",
```

Comment:
```suggestion
        ["shortName"] = "VOTI",
```

I imagine this was a typo?